TombEngine/Game/effects/effects.h

```diff
@@ -206,7 +208,7 @@ void TriggerCyborgSpark(int x, int y, int z, short xv, short yv, short zv);
 void TriggerExplosionSparks(int x, int y, int z, int extraTrig, int dynamic, int uw, int roomNumber);
 void TriggerExplosionSmokeEnd(int x, int y, int z, int uw);
 void TriggerExplosionSmoke(int x, int y, int z, int uw);
-void TriggerFireFlame(int x, int y, int z, FlameType type);
+void TriggerFireFlame(int x, int y, int z, FlameType type, Vector3 color1 = Vector3(1, 1, 1), Vector3 color2 = Vector3(1, 1, 1));
```

Comment:
You can simply use the `Vector3::One` constants here. They're pretty handy.

Reply:
ah, this is cool. I have changed it.